src/Kernel32/Kernel32+FileSystemFeature.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Copyright © .NET Foundation and Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+
+    /// <content>
+    /// Contains the <see cref="FileSystemFeature"/> nested type.
+    /// </content>
+    public partial class Kernel32
+    {
+        /// <summary>
+        /// Flags associated with a file system.
+        /// </summary>
+        [Flags]
+        public enum FileSystemFeature : uint
```

Comment:
That feels very odd. This value does not represent flags which influence the behavior of the function, but rather flags which provide information about features which are available on that filesystem. The parameter name is `lpFileSystemFlags`, so what about naming the enum `FileSystemFlags`?

Reply:
It's a tough thing giving a name to an enum when the headers don't have a name. Those "constants" that in C# we define as enum values could theoretically be used anywhere, in potentially arbitrary combinations. So needless to say, this isn't a hard and fast rule, so your suggestion may be appropriate. 
I imagine there may be other enums that we'll define for the file system though. So if we can think of a more precise name for this enum based on its content or where it's used, that's great. Otherwise, `FileSystemFlags` is probably OK.